packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -99,6 +99,7 @@ export const handleSDKReadyEvent = () =>  (
   dispatch: ThunkDispatch<void, any, Action>
 ) => {
   // wire up events for sdk
+console.log("In handleSDKReadyEvent");
```

Comment:
Is this console log needed for debugging whe app is running?

Reply:
Oops, no I meant to remove this one and just missed it.